js/app.js

```diff
@@ -1,32 +1,221 @@
-var warper = {
-  init: function() {
-    var endpoint = 'warper api endpoint';
-
-    var request = $.ajax({
-      url: endpoint,
-      method: 'GET'
-    });
-
-    request.done(function() {
-      // init
-    });
-
-    request.fail(function() {
-      warper.alert('negative', 'The Warper seams to be offline.');
-    });
-  },
-
-  alert: function(type, message) {
-    var validAlertTypes = ['progressive', 'negative', 'default'];
-    console.log(validAlertTypes.type);
-
-    if ($.inArray(validAlertTypes, type)) {
-      $('.alert-container').append('<div class="alert ' + type +'">' + message + '</div>');
-      setTimeout(function() {
-        $('.alert-container .alert').first().remove();
-      }, 6000);
-    } else {
-      console.log('invalid alert type');
-    }
-  }
-};
+// Leaflet integration for Map Warper (Wikimaps)
+
+
+var W_LEAFLET = {
+    map_div:		'map',
+    base_url:		'http://warper.wmflabs.org',
```

Comment:
They are the same but I'm using currently proxy for testing: http://wikihacks.opendimension.org/wmaps_html/ so we can have a working demo.